lib/analyzer/binary-version-extractors/openjdk-jre.ts

```diff
@@ -30,15 +30,19 @@ function parseOpenJDKBinary(fullVersionOutput: string) {
   */
   const jdkVersionLines = fullVersionOutput &&
                           fullVersionOutput.trim().split('\n');
-  if (!jdkVersionLines || jdkVersionLines.length !== 3) {
+  if (!jdkVersionLines) {
     return null;
   }
   const bracketsRE = /\(build (.*)\)$/;
-  const buildVersion = jdkVersionLines[1].match(bracketsRE);
-  const version = buildVersion && buildVersion[1];
+  const runtimeEnv = 'Runtime Environment';
+  const buildVersion =
+    (jdkVersionLines.filter(line => line.includes(runtimeEnv)).pop() || '')
+    .match(bracketsRE);
+  let version = buildVersion && buildVersion[1];
   if (!version) {
     return null;
   }
+  version = version.replace('-adoptopenjdk', '');
```

Comment:
My gut feeling is that we shouldn't remove the original version - but being able to parse it by the binary version translator...

Reply:
This is a little beyond me, @orkamara wdyt?

Reply:
Yes. Maybe it's a better one. Cause this way, we'll be able to display the users the actual versions. So they will see it's adopt and not a regular openjdk

Reply:
Sorry @leongold - can you revert it?